### ISSUE TITLE
Very Inaccurate Predictions 

### ISSUE THREAD

**Issue by shashi-netra** [inspirehep/magpie#149]
I trained a magpie model on an existing corpus (which we have trained & validated with other approaches such as xgboost). But the predictions from magpie appear to be very, very poor and I am not sure what I am doing wrong. Note: the corpus is from titles/descriptions of Youtube videos, we scrubbed it off stopwords and punctuation.

Here are some examples:

```
jackpot las vegas silverton casino jackpot people blogs los angeles dodgers
[('Publishing', 0.9999976), ('Events-Arts & Music', 0.9949007), ('Recreation-Museum', 0.14553918)]
```

```
legendary tv host john mclaughlin dead 89 john mclaughlin former jesuit priest nixon speech writer pioneered political talk show format mclaughlin group never missed broadcast past sunday mclaughlin 89 video cbs news john mclaughlin dead legendary political telivision host news politics cbs evening news
[('Recreation-Yoga', 0.9922431), ('Sports', 0.89598936), ('Sports-Hockey', 0.16487549)]
```

I have trained using default settings for 10 epochs

```
magpie.init_word_vectors('./data', vec_dim=100)
magpie.train('./data', labels, test_ratio=0.2, epochs=10)

```

**Comment by shashi-netra** [comment]
OK I finally figured this out and it was a silly mistake. I had the same label repeated multiple times in the `.lab` file. Just ensure a label appears only once in the `.lab` file.
